mautic/mautic
# Add integration relationships tab to contact details' page

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Related user documentation PR URL | N/A
| Related developer documentation PR URL | N/A
| Issues addressed (#s or URLs) | N/A
| BC breaks? | N/A
| Deprecations? | N/A

#### Description:
This PR creates a new tab beside Social in the contact details' page to display integration relationships based on values in the integration_entity table.

#### Steps to test this PR:
1. Apply this PR
2. Go to a contact details' page. You'll notice the new Integrations tab beside the Social Tab
3. Click it to view the integration relationships. If the contact doesn't have any, you can create one by pushing the contact to an integration. You can also fetch leads with the `app/console mautic:integration:fetchleads -i INTEGRATION` command.
